frontend/test/unit/specs/stores/search-store.spec.js

```diff
@@ -491,12 +491,12 @@ describe("Search Store", () => {
     `(
       "changing searchType from $searchType clears all but $expectedFilterCount $nextSearchType filters",
       async ({ searchType, nextSearchType, expectedFilterCount }) => {
-        const searchStore = useSearchStore()
-        searchStore.setSearchType(searchType)
-
         const featureFlagStore = useFeatureFlagStore()
         featureFlagStore.toggleFeature("additional_search_types", "on")
 
+        const searchStore = useSearchStore()
+        searchStore.setSearchType(searchType)
+
```

Comment:
Can you explain if the order is important here, and if so, can you document it in the file?